src/gui/inputdevices/PositionInputData.h

```diff
@@ -14,7 +14,9 @@
 
 #include <gtk/gtk.h>
 
-class PositionInputData {
+#include "util/Point.h"
+
+class PositionInputData: public utl::Point<double> {
```

Comment:
Do not use inheritance here:
 1. `utl::Point<>` does not provide a virtual destructor, which will introduce unnecessarily a v-table for such a trivial struct.
 2. `PositionInputData` is not a Point it contains one. So make `utl::Point<double>` a member with the name `pos` or `position`. A doc is not required, since this is self explaining.